AirCasting/MainTabBarView.swift

```diff
@@ -84,6 +84,7 @@ private extension MainTabBarView {
 
 class TabBarSelection: ObservableObject {
     @Published var selection = Tab.dashboard
+    @Published var mobileProceeding = false
```

Comment:
what it stands for actually? I can't figure it out. Name is unclear or it doesn't belong here. `TabBarSelection` should only be about tab bar selection ;)

After reading the trello I'm quite convinced this is not the way it should be handled. Consider combining the value of `TabBarSelection` value with the `AirSectionPickerView.SelectedSection` to recognize which flow the app should take when create is pressed instead of this flag. Do it in appropriate layer. And don't be afraid of just creating new objects if you feel like something doesn't fit anywhere.

Reply:
Yeah, just realise we have the ability to use something like this

Reply:
Cool. thank You!